src/scripts/destinyTrackerApi/reviewDataCache.js

```diff
@@ -0,0 +1,86 @@
+import _ from 'underscore';
+import { itemTransformer } from './itemTransformer.js';
+
+class reviewDataCache {
+  constructor() {
+    this._itemTransformer = new itemTransformer();
+    this._itemStores = [];
+  }
+
+  getMatchingItem(item) {
+    var dtrItem = this._itemTransformer.translateToDtrWeapon(item);
+
+    dtrItem.referenceId = String(dtrItem.referenceId);
```

Comment:
It's an inconsistency with how the API returns reference ID between calls.

Or at least did.  I let them know, but treating them all as strings fixed some mismatches on our end.